### PR TITLE
fix #35 - update SUSI API

### LINKED ISSUE #35: Update SUSI server API
**Actual Behaviour**

The current API is `http://api.asksusi.com/susi/chat.json` which is obsolete.

**Expected Behaviour**

The correct API should be `https://api.susi.ai/susi/chat.json`.

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by hkedia321**
Fixes issue #35 

Changes: 
- update SUSI server API.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ bot.onText(/\/start/, function (msg, match) {
 // Any other message
 bot.on('message', function (msg) {
 	var chatId = msg.chat.id;
-	var queryUrl = 'http://api.asksusi.com/susi/chat.json?q='+encodeURI(msg.text);
+	var queryUrl = 'http://api.susi.ai/susi/chat.json??q='+encodeURI(msg.text);
```
Comment (DNS-404) [inline app.js]: Two "?" after `chat.json`?

**Comment by hkedia321** [comment]
@DNS-404 corrected!

**Comment by AliAyub007** [comment]
@hkedia321 Kindly fix build failures.

**Comment by hkedia321** [comment]
@AliAyub007 fixed the codacy failures :+1:

**Pull request merged**